src/wp-includes/script-loader.php

```diff
@@ -113,9 +113,9 @@ function wp_default_packages_vendor( $scripts ) {
 		'lodash'                      => '4.17.19',
 		'wp-polyfill-fetch'           => '3.6.17',
 		'wp-polyfill-formdata'        => '4.0.10',
-		'wp-polyfill-node-contains'   => '4.6.0',
+		'wp-polyfill-node-contains'   => '4.7.0',
```

Comment:
This should be updated to match the package version, `4.8.0`. Seems this was missed on the last update.a

Reply:
@desrosj  "wp-polyfill-node-contains" updated to 4.8.0 and run `npm install ` command.